### DISCUSSION TITLE
POLL: Issues with Google Photos API

### DISCUSSION THREAD

**Discussion by gilesknap** [gilesknap/gphotos-sync#359] [Polls]
The Google Photos API has a number of issues which limit the ability to extract your data from the Photos Library. See #119.
These include:

- No GPS data in any images
- All videos downsampled to a poor quality
- All raw photos converted to jpg
- All Original Quality photos also down sampled (to High Quality)
- No meta data regarding updates to the photos (requires a full rescan to detect modified photos/videos)
- burst shots not available via API
- previous versions of files not available via API

All of the above are not issues if you use the Photos Web UI, only when going via Google Photos REST API.

Join the discussion on theses issues here #347

**Comment by kaspy003** [comment]
Yes google, wtf? - All Original Quality photos also down sampled (to High Quality) I build my photographer web app with google photos integration but no original photo download option? almost 3 years!!!!!!
